test/sign.spec.ts

```diff
@@ -23,21 +25,21 @@ export async function testSign(command: string): Promise<`0x${string}`> {
 
 export async function testSignCLIPrivateKey(data: string): Promise<`0x${string}`> {
   return testSign(
-    "npm run cli sign ethereum 0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133 " +
+    "npm run cli sign --type ethereum --privateKey 0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133 " +
```

Comment:
@crystalin I don't understand why if I append the last argument (message) with `--message`, it says "one argument missing"

Reply:
don't you need `--` to pass the argument to the script (instead of passing them to npm) ?

Reply:
Apparently not. maybe because Im using yarg's command

Reply:
Oh okay I have figured it out: `--` is indeed needed but I needed to clear the command/usage first

Reply:
Oh okay I have figured it out: `--` is indeed needed but I needed to clear the command/usage first